.github/workflows/test_sllm_store.yaml

```diff
@@ -0,0 +1,51 @@
+name: Test Sllm Store
+
+on: workflow_dispatch
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'  # Specify the Python version you want to use
+
+      - name: Free disk space
+        run: |
+          df -h
+          sudo rm -rf /usr/local/cuda-* /opt/cuda
+          sudo rm -rf /usr/local/cuda
+          bash -x .github/workflows/scripts/free-disk-space.sh
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements-test.txt
+
+      - name: Install CUDA 12.1
+        run: |
+          bash -x .github/workflows/scripts/cuda-install.sh 12.1 ubuntu-22.04
+
+      # Python Tests
+      - name: Install ServerlessLLM Store
+        run: |
+          export TORCH_CUDA_ARCH_LIST="8.0 8.6 8.9 9.0"
+          cd serverless_llm/store && pip install .
+
+      - name: Run Python tests
+        run: |
+          pytest serverless_llm/store/tests/python
+
+      # C++ Tests
+      - name: Build C++ project
+        run: |
+          export TORCH_CUDA_ARCH_LIST="8.0 8.6 8.9 9.0"
+          cd serverless_llm/store && pip install -r requirements-build.txt && bash cpp_build.sh
```

Comment:
It seems that the C++ tests and python tests share the same environment. I think it may be better if we could divide them into two different workflows?

Reply:
So actually the python test doesn't rely on GPu and the C++ test rely on GPU. Thus I suppose move them into two diffferent workflows, and one could be automatically run on github actions while push / pull request and another could be manually triggered in order to save GPU usage.

Reply:
I intend to have the C++ and Python tests share the same environment because: 1) they can use a single CUDA installation, reducing setup time; and 2) both components are meant to work together in the same environment. If they only pass tests in separate environments, that would indicate an underlying issue.